detectable/src/main/java/com/blackduck/integration/detectable/python/util/PythonDependencyTransformer.java

```diff
@@ -109,6 +108,9 @@ public List<Object> findOperatorWithIndex(String line) {
     }
 
     public String formatLine(String line) {
+        if (line.contains("\u0000")) {
```

Comment:
Since we are not using a regex, would replace(...) be better?
Or, if the length of the String and/or the number of substrings to replace are unknown/large, could we use StringBuilder?